lib/build-tokens.js

```diff
@@ -139,6 +139,14 @@ async function buildTokensCommand(commandArgs) {
               outputReferences,
             },
           },
+          {
+            format: 'css/button-variant-overrides-alert-actions',
+            destination: `themes/${themeVariant}/overrides/alert.css`,
```

Comment:
Similar to above, this proposed putting all generated, component-specific overrides under an `overrides` directory (e.g., this might include `/overrides/modal.css` in the future).